### DISCUSSION TITLE
Carbon's 2023 roadmap and a look back at 2022

### DISCUSSION THREAD

**Discussion by chandlerc** [carbon-language/carbon-lang#2553] [Announcements]
Hello everyone,

We've just landed our [2023 Roadmap](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/roadmap.md) update, as well as a look back at [how 2022 went for the project](https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p2551.md#retrospective-on-2022).

Check both of these out, and we're happy to discuss these here. Looking forward to 2023 and everything coming up for the Carbon Language project!

**Comment by ki37ki** [comment]
Nice 👌
